docs/manager.md

```diff
@@ -89,18 +89,18 @@ Manager are entitled to collect fees over the investments to insure longetivity
 
 ### Management fee
 
-A fixed 1% management fees has been set in the smart contracts which managers can collect when they move the funds from router to their fund contracts. This amount will be shown on manager dashboard interface. 
+A fixed 1% management fees has been set in the smart contracts which managers can collect when they move the funds from router to their fund contracts. This amount is intended to incentivise the Manager for the costs borne for transaction fees and can be collected from the Manager Dashboard. 
```

Comment:
A fixed 1% management fees has been set in the smart contracts which managers can collect when they move the funds from the initial router vault to their fund contracts. This amount is intended to compensate the Manager for the costs borne from transaction fees. This fee can be collected from the Manager Dashboard.